moler/config/loggers.py

```diff
@@ -27,6 +28,46 @@
 raw_logs_active = False
 write_mode = "a"
 
+moler_logo = """
+                        %%%%%%%%%%%%%%%%%%%%%
+                   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+                %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+             %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+           %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+         %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+       %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+      %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  %%%%%%%%%%%%
+     %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%    %%%%%%%%%%%%
+    %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%      %%%%%%%%%%%%%
+   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%        %%%%%%%%%%%%%%%
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%                     %%%%%%%%%%%%%%%%
+  %%%%%%%%%%%%%%%%%%%%%                          %%%%%%%%%%%%%%%%%%
+ %%%%%%%%%%%%%%%%%%%%                           %%%%%%%%%%%%%%%%%%%
+ %%%%%%%%%%%%%%%%%%                            %%%%%%%%%%%%%%%%%%%%%
+ %%%%%%%%%%%%%%%%       €€                    %%%%%%%%%%%%%%%%%%%%%%
+ %%%%%%%%%%%%%%%                            %%%%%%%%%%%%%%%%%%%%%%%%
+ %%%%%%%%%%%%%%                          %%%%, %%%%%%%%%%%%%%%%%%%%%
+ %%%%%%%%%%%%%                                %%%%%%%%%%%%%%%%%%%%%
+  %%%%%%%%%%%%                              %%%%%%%%%%%%%%%%%%%%%%%
+  %%%%%%%%%%%                               %%%%%%%%%%%%%%%%%%%%%%
+   %%%%%%%%%                                   %%%%%%%%%%%%%%%%%%%
+    %%%%%%%%                                     %%%%%%%%%%%%%%%%
+     %%%%%%    https://github.com/nokia/moler     %%%%%%%%%%%%%%
+
+    %%%%     %%%%   %%%%%%%%%%   %%%       %%%%%%%%%% %%%%%%%%%%
+    %%%%%   %%%%%  %%%       %%  %%%       %%%        %%%     %%%
+    %% %%% %%% %%  %%%       %%  %%%       %%%%%%%%%  %%% %%%%%%%
+    %%  %%%%%  %%  %%%       %%  %%%       %%%        %%%   %%%
+    %%   %%%   %%   %%%%%%%%%%%  %%%%%%%%% %%%%%%%%%% %%%     %%%
+"""
+
+
+def _get_moler_version():
+    try:
+        return pkg_resources.get_distribution("moler").version
+    except pkg_resources.DistributionNotFound:
+        return "cloned from git repository"
```

Comment:
if it is from git repo you can also retrieve version:
```python
import os
setup_py_path = os.path.join(os.path.dirname(__file__), "..", "..", "setup.py")
```
try if file exists, open it and find version

Reply:
something like that?